core/src/main/java/org/jdbi/v3/core/statement/ParsedSql.java

```diff
@@ -72,6 +72,25 @@ public String toString() {
             + '}';
     }
 
+    /**
+     * A static factory for {@link ParsedSql} object. It takes two
+     * arguments. The first one is SQL which must contain only
+     * positional parameters (i.e. the {@value #POSITIONAL_PARAM}
+     * characters). If your code contains named parameters (i.e.
+     * the ones followed by a colon) then you have to replace them
+     * by positional parameters and store mapping as
+     * {@link ParsedParameters} which is taken as the second
+     * argument. Please also note that you cannot mix named and
+     * positional parameters in SQL statement.
+     *
+     * @param sql the sql code containing only positional parameters
+     * @param parameters the list of named parameters (ordered)
```

Comment:
", or an appropriate positional ParsedParameters object"

Reply:
Will change to "the ordered list of named or positional parameters". Is this ok?

Reply:
"the ordered list of named parameters, or positional parameters" then, to avoid dubious grammar

Reply:
Done.